### PR TITLE
Add ember test globals

### LINKED ISSUE #42: Add Ember.Test methods
I was wondering if the [Ember.Test](http://emberjs.com/api/classes/Ember.Test.html) methods could be added to the globals. I'm currently manually adding them to my `globals` in `tests/.eslintrc` for an ember-cli project. But it would be nice to just use an `env` setting. Would you accept a pull request for them?

Related to: https://github.com/eslint/eslint/issues/2870

### PR DESCRIPTION
**Pull request by ismay**
Since the entries weren't sorted alphabetically I've just added the globals at the end. They're under the `ember` key,  but I can change that to `ember.test` if you'd like. Let me know if there's anything that should be changed!

Closes #42

### CODE REVIEW

**Comment by sindresorhus** [comment]
`ember` is too ambiguous. `ember-test`?

File: globals.json [hunk 1]
```diff
@@ -970,5 +970,17 @@
...
+		"click": false,
+		"fillIn": false,
+		"keyEvent": false,
+		"triggerEvent": false,
+		"visit": false,
+		"currentPath": false,
+		"currentRouteName": false,
+		"currentUrl": false,
+		"find": false,
+		"andThen": false
```
Comment (sindresorhus) [inline globals.json]: These keys in the "ember" object should be sorted alphabetically.

**Comment by ismay** [comment]
Yeah `ember-test` sgtm. I'll change it and sort the keys.

**Comment by ismay** [comment]
Want me to squash the commits, or is this fine?

**Comment by sindresorhus** [comment]
I can squash when I merge.

Gonna give @nzakas a chance to review as I'm not sure whether he prefers it to be `ember-test` or `emberTest`.

**Comment by nzakas** [comment]
I'd go with `embertest`, which would match `serviceworker`. Thoughts?

**Comment by sindresorhus** [comment]
In hindsight it should been `service-worker` or `serviceWorker`, but I guess we can go for consistency.

**Comment by ismay** [comment]
Ok, amended.

**Pull request closed without merging**